.github/workflows/test.yml

```diff
@@ -68,7 +68,19 @@ jobs:
 
       - name: Run unit tests
         id: tests
-        run: pytest --junit-xml=test-results.xml --durations=25
+        run: |
```

Comment:
We probably should run the UI tests in a different job to be able to parametrize over the browser. I would test Chromium and Firefox on every operating system and additionally Safari (WebKit) on macOS. As for the Python version, I'd say we use the same as we have for docker, i.e.

https://github.com/Quansight/ragna/blob/a45bd9063f106a7728c5fb65c9077387b25c9706/Dockerfile#L1

Meaning, we'll have

```yaml
strategy:
  matrix:
    os:
      - ubuntu-latest
      - windows-latest
      - macos-latest
    browser:
      - chromium
      - firefox
    include:
      - os: macos-latest
        browser: webkit
```

The test command for the "regular" tests will change to `pytest --ignore tests/ui` and for UI tests `pytest tests/ui --browser ${{ matrix.browser }}`.